backend/config/initializers/devise.rb

```diff
@@ -314,9 +314,16 @@
   # config.sign_in_after_change_password = true
 
   config.jwt do |jwt|
-    jwt.secret = Rails.application.credentials.jwt_secret_key
+    jwt.secret = ENV['JWT_SECRET_KEY'] || Rails.application.credentials.jwt_secret_key
```

Comment:
Added this so that you can pop `JWT_SECRET_KEY` in a .env file set to some nonsense and avoid having to have the master key to decrypt the credential store. This was blocking the server from actually making the JWTs